contracts/interfaces/IGetters.sol

```diff
@@ -43,7 +43,7 @@ interface IGetters {
 
     /// @notice Returns the collateral ratio of Transmuter in base `10**9` and a rounded version of the total amount
     /// of stablecoins issued
-    function getCollateralRatio() external view returns (uint64 collatRatio, uint256 stablecoinsIssued);
+    function getCollateralRatio() external returns (uint64 collatRatio, uint256 stablecoinsIssued);
```

Comment:
Are we sure about the need to protect the getCollateralRatio function from reentrancy call?

Reply:
can just leave it as view and check if the system is in an entered state in which case it should revert